polaris-ui/src/app/features/cases/presentation/case-details/index.tsx

```diff
@@ -236,6 +241,18 @@ export const Page: React.FC<Props> = ({ backLinkProps, context }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [accordionState.status]);
 
+  useEffect(() => {
+    if (usedOrUnused.saveStatus === "initial") {
+      setSavingStateDescrition("initial");
```

Comment:
i think its better to pass in usedOrUnused.saveStatus directly without creating a new state , savingStateDescrition